backoff_tests.py

```diff
@@ -196,7 +319,7 @@ def succeeder(*args, **kwargs):
                            'kwargs': {'foo': 1, 'bar': 2},
                            'target': succeeder._target,
                            'tries': i + 1,
-                           'wait': 0}
+                           'wait': 1}
```

Comment:
With jitter=None above, no need to change this.

Let me know if I'm missing something.